examples/generate-python-complete-models/index.ts

```diff
@@ -0,0 +1,37 @@
+import { PythonGenerator } from '../../src';
+import { PythonOptions } from '../../src/generators/python/PythonGenerator';
+import { DeepPartial } from '../../src/utils';
+
+let generator: PythonGenerator;
+export const jsonSchemaDraft7 = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    email: {
+      type: 'string',
+      format: 'email'
+    },
+    objProperty: {
+      type: 'object',
+      properties: {
+        number: {
+          type: 'number'
+        }
+      }
+    }
+  }
+};
+
+export async function generate(options: DeepPartial<PythonOptions>) : Promise<void> {
+  generator = new PythonGenerator(options);
+  const models = await generator.generateCompleteModels(jsonSchemaDraft7, options);
+  for (const model of models) {
+    console.log(model.result);
+  }
+}
+
+// if (require.main === module) {
+//   generator = new PythonGenerator(PythonGenerator.defaultOptions);
+//   generate({});
+// }
```

Comment:
Oh, I committed a commented `if (require.main === module) {` that part by a mistake. Let's just uncomment it so the script would run as `start`, at the same time, tests will be more granular: two separate test items for `implicit` and `explicit` cases, what do you think?

Reply:
Restored the `npm run start` functionality, hope that will do the trick.

Reply:
@zaytsevand I dont see any new commits? 🙂

Reply:
@jonaslagoni oh, sorry. I haven't checked the command output 🙈 
```
git push
ssh: Could not resolve hostname github.com: Temporary failure in name resolution
fatal: Could not read from remote repository.
```
I would love to have the tests more granular (i.e. one for `explicit` and one for `implicit`) and to support `npm run start` by passing default options, as committed. But if you see the future development of test cases otherwise then np :)